packages/angular-generator/src/expressions/jsx/ng-content-generator.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Component,
+  Identifier,
+  JsxAttribute,
+  JsxSpreadAttribute,
+  SimpleExpression,
+} from '@devextreme-generator/core';
+import { AngularDirective } from './angular-directive';
+import type { JsxOpeningElement } from './jsx-opening-element';
+import { getUniqComponentName } from '../utils/uniq_name_generator';
+
+export function tryToGetContent(element: JsxOpeningElement): {
+  content: string;
+  elementDirective: AngularDirective | null;
+  condition: AngularDirective | null;
+} {
+  const componentName = element.tagName.toString();
+  const isComponent = element.context.components?.[componentName];
+  let elementDirective = null;
+  let content = '';
+
+  if (!isComponent || (isComponent as Component).isSVGComponent) {
+    return { content, elementDirective, condition: null };
+  }
+
+  const refAttr = element.attributes.find((attr: JsxAttribute | JsxSpreadAttribute) => attr.toString()[0] === '#');
+  let ref = refAttr?.toString().split('.').pop()?.replace('#', '');
+  if (!ref) {
```

Comment:
Fot this you can try using JsxAttribute.isRefAttribute and getRefValue functions, same for next lines - try to use already defined methods as max as possible, because your code can work correctly, but potentially contains bugs